examples/AffinePlaneCurve.jl

```diff
@@ -213,12 +61,12 @@ end
 
 Return the affine plane curve consisting of the common component of `C` and `D`, or an empty vector if they do not have a common component.
 """
-function common_component(C::AffinePlaneCurve{S}, D::AffinePlaneCurve{S}) where S <: FieldElem
+function common_components(C::AffinePlaneCurve{S}, D::AffinePlaneCurve{S}) where S <: FieldElem
   G = gcd(C.eq, D.eq)
   if isone(G)
      return Vector{AffinePlaneCurve}()
   else
-     return AffinePlaneCurve(G)
+     return Vector([AffinePlaneCurve(G)])
```

Comment:
`[...]` is already a `Vector` constructor:
```suggestion
     return [AffinePlaneCurve(G)]
```